src/test/regress/citus_tests/common.py

```diff
@@ -93,6 +93,7 @@ def get_pg_major_version():
     14: "10.2.0",
     15: "11.1.5",
     16: "12.1devel",
+    17: "12.1devel",
```

Comment:
The current version on main branch as well as this feature branch is `12.2devel`. We no longer care about `12.1devel` at all.

```suggestion
    16: "12.1.0",
    17: "12.2devel",
```

Reply:
Makes sense. I will fix that, but not in this PR, because the first commit of this PR is not to be merged.